scripts/onboarding/templates/arc-k8s-extension-msi-auth/existingClusterOnboarding.json

```diff
@@ -260,8 +259,8 @@
       "type": "Microsoft.Resources/deployments",
       "name": "[Concat('arc-k8s-ci-extension', '-',  uniqueString(parameters('clusterResourceId')))]",
       "apiVersion": "2019-05-01",
-      "subscriptionId": "[split(parameters('clusterResourceId'),'/')[2]]",
-      "resourceGroup": "[split(parameters('clusterResourceId'),'/')[4]]",
+      "subscriptionId": "[variables('clusterSubscriptionId')]",

```

Comment:
I see the same template changes in both PRs and you can revert the changes in this PR and keep other one.